### PR TITLE
autopop mntnr info; `eris known, ls`; pull images on init; closes #68, #91, #112

### LINKED ISSUE #68: auto populate maintainer information on `chains new` and `services new`
use git config to populate.

### PR DESCRIPTION
**Pull request by zramsay**

### CODE REVIEW

File: util/init.go [hunk 1]
```diff
@@ -28,6 +28,38 @@ func Initialize(toPull, verbose bool) error {
...
+	fmt.Println("Pulling base images...")
+
+	if !dev {
+		argsAll = argsDef
+	} else {
+		fmt.Println("...and development images")
+		argsAll = append(argsDef, argsDev...)
+	}
+
+	fmt.Println("This could take awhile, now is a good time to feed your marmot")
```
Comment (compleatang) [inline util/init.go]: :+1:

File: util/init.go [hunk 2]
```diff
@@ -28,6 +28,38 @@ func Initialize(toPull, verbose bool) error {
...
+	for _, img := range argsAll {
+		pImg := []string{"pull", img}
+		err := exec.Command("docker", pImg...).Run()
```
Comment (compleatang) [inline util/init.go]: Let us use the `perform.DockerPull` function to do this via the API (which will be more controllable for us than using os/exec) its [here](https://github.com/eris-ltd/eris-cli/blob/develop/services/operate.go#L137).  Basically just need something like:

```go
srv := definitions.BlankService()
srv.Image = img
ops := definitions.BlankOperation()
err := perform.DockerPull(srv, ops)
```

**Comment by compleatang** [comment]
This is ready once you are able to make that one change noted above. Feel free to make that and then pull it in. :+1:

File: util/init.go (hunk 2 above)
Comment (zramsay) [inline util/init.go]: perform imports util in a non-trival way...this won't fly

**Comment by zramsay** [comment]
should we silence the failing test on circle (see #containers)? other than that I think it's ready to go

**Pull request merged**